pkg/bytecode/compiler.go

```diff
@@ -57,10 +60,14 @@ func (c *Compiler) Compile(node parser.Node) error {
 		return c.compileAssignment(node)
 	case *parser.BinaryExpression:
 		return c.compileBinaryExpression(node)
+	case *parser.BreakStmt:
+		return c.compileBreakStatement(node)
 	case *parser.BlockStatement:
 		return c.compileBlockStatement(node)
 	case *parser.IfStmt:
 		return c.compileIfStatement(node)
+	case *parser.WhileStmt:
+		return c.compileWhileStatement(node)
```

Comment:
worth mentioning that Thorsten's Monkey language doesn't support loops so we're off-book for this